.eslintrc.json

```diff
@@ -52,22 +52,35 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": [
-          "**/stories/**/*.*",
-          "**/.storybook/**/*.*"
-        ],
+        "devDependencies": ["**/stories/**/*.*", "**/.storybook/**/*.*"],
         "peerDependencies": true
       }
     ]
   },
   "overrides": [
     {
-      "files": "tests/**/*.ts",
+      "files": "**/*.test.{ts,tsx,js,jsx}",
       "env": {
         "node": true,
-        "jest": true
+        "jest": true,
+        "jest/globals": true
+      },
+      "plugins": ["jest", "no-snapshot-testing", "testing-library"],
+      "extends": [
+        "airbnb-typescript",
+        "prettier",
+        "plugin:@cspell/recommended",
+        "plugin:jest/recommended",
+        "plugin:testing-library/react"
+      ],
+      "rules": {
+        "testing-library/no-node-access": ["error", { "allowContainerFirstChild": true }],
```

Comment:
this is optional but I'd like if we added links to some of the eslint rule for others to know what they mean would be great.

I grabbed the link for you already 
- https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-node-access.md

Reply:
[d4ee190](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/175/commits/d4ee1905b6eee8d21815a55f2716817ea09ce939)